### ISSUE TITLE
Multiple characters in a label cause a segfault

### ISSUE THREAD

**Issue by lorenlugosch** [parlance/ctcdecode#71]
When I try to use a multiple-character string as a label, I get a segfault. 
For example:
```
import torch
import ctcdecode
labels = ["_", "SIL", "A"]
decoder = ctcdecode.CTCBeamDecoder(labels, blank_id=0)
decoder.decode(torch.randn(3,3,3))
```
triggers a segfault, whereas 
```
import torch
import ctcdecode
labels = ["_", "A", "B"]
decoder = ctcdecode.CTCBeamDecoder(labels, blank_id=0)
decoder.decode(torch.randn(3,3,3))
```
does not. Is it possible to do this somehow?

**Comment by ryanleary** [comment]
Not today, see #31.
